config/default.json

```diff
@@ -267,6 +267,7 @@
       "defaultLocale": "en-US",
       "currencyCode": "USD",
       "currencySign": "$",
```

Comment:
Can we rather use the naming like:
currencySign: “append”/“prepend”? Plus add a line into config docs and let know @NataliaTepluhina to update the official docs :)

Reply:
Not sure exactly what to change it to, @pkarw you mean `"currencySignAppend": false` ?

Reply:
I was rather up to changing this from bool to string (well - enum kind of with “append” and “preppend” options)

Reply:
To something like `currentSignPlacement: 'append' || 'prepend'` ? Or how would you @pkarw name the option "currentSignPlacement" ? :)

Reply:
Correct, i was thinking about something like this. I’m not a naming-nazi ;) so please use the naming of your choice it was more like suggestion / idea :-)